webui/react/src/shared/bin/test.py

```diff
@@ -6,7 +6,6 @@
 
```

Comment:
this and the readme change jsut revert changes that probably should have been added to shared-web first?